internal/cluster/cluster.go

```diff
@@ -290,6 +290,86 @@ type ClusterSpec struct {
 	AutomaticPgRestart *bool `json:"automaticPgRestart"`
 }
 
+type ClusterSpecNew struct {
```

Comment:
There's the need of two different cluster spec structs identical to the original cluster spec:

* one with all json tags with omitempty for normal output
* one with all json tags without omitempty for --default output